go.mod

```diff
@@ -29,6 +29,7 @@ require (
 	github.com/influxdata/usage-client v0.0.0-20160829180054-6d3895376368
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/jonboulle/clockwork v0.1.0 // indirect
+	github.com/kevinburke/go-bindata v3.16.0+incompatible // indirect
```

Comment:
we probably should just leave this in, as they get injected every time we run `make`, but it's not used by chronograf outside the makefile. to avoid churn, remove these `go.*` changes either manually or with a `go mod tidy`.